tests/openvino/native/test_node_utils.py

```diff
@@ -50,3 +55,34 @@ def test_is_node_with_bias(model_to_create, is_with_bias, node_name):
     nncf_graph = GraphConverter.create_nncf_graph(model)
     node = nncf_graph.get_node_by_name(node_name)
     assert is_node_with_bias(node, nncf_graph) == is_with_bias
+
+
+@pytest.mark.parametrize(
+    "weights_port_id, transpose, shape, expected_channel_axes",
+    [
+        (0, False, (1,), [0]),
+        (0, True, (1,), []),
+        (1, False, (1,), []),
+        (1, True, (1,), [0]),
```

Comment:
Why transpose in this cases is making different results? Is transpose of a scalar do something?

Reply:
As far as I understand, channel axes are output dimensions and "batch" dimensions, how it is possible for 1d case to have output dim? Does it always withdraw after the multiplication?